### ISSUE TITLE
README.txt on readthedocs

### ISSUE THREAD

**Issue by JoelPasvolsky** [dwavesystems/penaltymodel#19]
* Hard to follow 
* Would be helpful to open with something like this here: “A penalty model is a small Ising problem or QUBO that has ground states that match the feasible configurations and excited states that have a classical energy greater than the ground energy by at least the classical gap.”

**Comment by JoelPasvolsky** [comment]
Updated text: "One approach to solve a constraint satisfaction problem (CSP) using an Ising model or a QUBO, is to map each individual constraint in the CSP to a 'small' Ising model or QUBO. This mapping is called a penalty model."

I find the "using" ambiguous. Would it be correct to rephrase to something like:

"One approach to solving a constraint satisfaction problem (CSP) is to represent each constraint as a penalty. These penalties can be formulated as 'small' Ising models or QUBOs. This mapping is called a penalty model."

**Comment by arcondello** [comment]
@JoelPasvolsky  Take a look at the 1.0.0 branch README.rst (it is the same for the readthedocs) and let me know what you think

**Comment by JoelPasvolsky** [comment]
Very helpful!
Here are some minor comments:

•	Can we have the first sentence stay with the previous; i.e., follow the intro's “One approach to solve a … CSP using … a QUBO,” with a Boolean constraint rather than gate? So maybe, “…we want to map an ^ (AND) clause to a QUBO” just to reduce mixing terminiology
•	“feasible configurations that we wish to target”: can we add in parentheses something like “variable values that result in low energy”? Because this is introduces the term “feasible configurations” that not all users might know. 
•	“add an edge between each node, represnting”: between nodes, representing. It looks like we add a complete graph (i.e.,  between each node and all other nodes). Is that always the case or just here, in which case we "add edges between nodes that interact"? 
•	“So we encode this energy function as a QUBO”: be helpful here to say why the offset is 0, it’s the only thing that doesn’t come from the equation directly.
